### DISCUSSION TITLE
Go-E Charger

### DISCUSSION THREAD

**Discussion by MrGreenGuyDK** [goecharger/go-eCharger-API-v2#213] [General]
Hi Guys - im not able to get this working :-( - Home Assistant  all sensores are unavailable - i've followed the guides and still nothing - please support me

**Comment by marq24** [comment]
as far as I know, there is no HA integration supported officially - so you IMHO you can't expect to receive detailed support here (but I might be wrong).

There are current 2 integrations available via HACS - one of them requires the installation of an additional MQTT Server - that might be the reason why you do not see any sensor data... If you are in an experimental mood you might like to try another Integration alternative (that require to add a custom repository to HACS) -> details can be found here: https://github.com/marq24/ha-goecharger-api2

**Reply by 0xFEEDC0DE64** [reply]
actually we are working on a common homeassistant integration, it has already been released with the go-eController firmware, I am not sure if the go-eCharger app already shows fields to enable it.

The relevant api keys on go-eCharger and go-eController are:
mht: optional&lt;string&gt; MQTT homeassistant topic prefix (set to null to reset back to the default)
mhe: bool MQTT enable homeassistant discovery

I am also not completely sure if changing the parameters does require a MQTT reconnect or maybe even a charger / controller reboot

Let me know if you want to test it :)

**Comment by marq24** [comment]
> actually we are working on a common homeassistant integration...

Looks like that then I just was not patient enough... Since I just completed the development of a HA Integration supporting 145 of the 175 documented API-keys based on local HTTP API v2 [without the requirement for an additional MQTT instance (even if I have one instance running)]. That's also the reason why I spot the typos and created the two PR's just just merged :-)
